<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YoutubePreferences</class>
 <widget class="QDialog" name="YoutubePreferences">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupYoutubeOptions">
     <property name="title">
      <string>YouTube Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="textYoutubeKey">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelYoutubeKey">
        <property name="text">
         <string>&amp;YouTube API Key</string>
        </property>
        <property name="buddy">
         <cstring>textYoutubeKey</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelQuality">
        <property name="text">
         <string>&amp;Quality</string>
        </property>
        <property name="buddy">
         <cstring>comboQuality</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboQuality">
        <item>
         <property name="text">
          <string>High</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Low</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkPlayVideo">
        <property name="text">
         <string>Play Video</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSave">
     <property name="title">
      <string>Save in Computer</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSaveMusic">
        <property name="text">
         <string>Save &amp;Music</string>
        </property>
        <property name="buddy">
         <cstring>comboSaveMusic</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboSaveMusic">
        <item>
         <property name="text">
          <string>Don't save</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Save only after complete download</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Save while downloading</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSaveLocation">
        <property name="text">
         <string>Save &amp;Location</string>
        </property>
        <property name="buddy">
         <cstring>textSaveLocation</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="textSaveLocation"/>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSaveLocation">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonOKCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOKCancel</sender>
   <signal>accepted()</signal>
   <receiver>YoutubePreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOKCancel</sender>
   <signal>rejected()</signal>
   <receiver>YoutubePreferences</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
